pkg/apis/pipeline/v1alpha1/task_types.go

```diff
@@ -124,7 +119,6 @@ type Inputs struct {
 type TaskResource struct {
 	ResourceDeclaration `json:",inline"`
 	// +optional
-	OutputImageDir string `json:"outputImageDir,omitempty"`
```

Comment:
This is an API change, right?  Would it be worth supporting this field in a deprecated state for a version?

Reply:
It's technically part of the API, but I think it's unused. You can see that this value is never copied from the resource params into the resource in the constructor in image_resource.go.

Reply:
This is causing a issue, because we do have resources with `outputImageDir` and they cannot be updated now :(